static/js/controllers/reports.js

```diff
@@ -131,34 +187,60 @@ var _ = require('underscore'),
           return DB.get()
             .get(report)
             .then(FormatDataRecord);
-        } else {
-          return FormatDataRecord(report);
         }
+        return FormatDataRecord(report);
       };
 
       $scope.refreshReportSilently = function(report) {
-        _fetchFormattedReport(report)
+        return _fetchFormattedReport(report)
           .then(function(doc) {
-              _setSelected(doc[0]);
-            })
+            _setSelected(doc[0]);
+          })
           .catch(function(err) {
             $log.error('Error fetching formatted report', err);
           });
       };
 
+      $scope.deselectReport = function(report) {
+        for (var i = 0; i < $scope.selected.length; i++) {
+          if ($scope.selected[i]._id === report._id) {
+            $scope.selected.splice(i, 1);
+          }
+        }
+        $('#reports-list li[data-record-id="' + report._id + '"] input[type="checkbox"]')
```

Comment:
This is the only place I can see that checks by ID so I don't think this would gain anything. I'd love to use angular for everything but anything in the LHS has to be manually managed...